src/firewall.c

```diff
@@ -115,7 +115,8 @@ int pg_firewall_rule_add(struct pg_brick *brick, const char *filter,
 	g_assert(rule);
 	npf_rule_setprio(rule, NPF_PRI_LAST);
 	if (filter && firewall_build_pcap_filter(rule, filter)) {
-		*errp = pg_error_new("pcap filter build failed");
+		*errp = pg_error_new("pcap filter build failed: %s",
```

Comment:
here you printing the filter, but the sentence sound you will give the reason of the failure.
I would rather use this: `error_new("pcap filter build failed with this filter: '%s'", filter`